_docs/drivers.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: page
+title: Drivers
+permalink: /docs/drivers/
+---
+
+As described in the [Automation Pipeline](../docs/overview) document, a driver
+must be present on the same machine as the browser you are automating.
+
+If you are running locally, you can use the [webdrivers gem](https://github.com/titusfortner/webdrivers)
+to automatically ensures that the latest driver is downloaded, and placed in a location
```

Comment:
I think this should be "that automatically ensures" or "to automatically ensure".